packages/flutter_tools/lib/src/commands/init.dart

```diff
@@ -201,14 +201,27 @@ void main() {
   runApp(
     new MaterialApp(
       title: "Flutter Demo",
-      routes: {
+      routes: <String, RouteBuilder>{
         '/': (RouteArguments args) => new FlutterDemo()
       }
     )
   );
 }
 
-class FlutterDemo extends StatelessComponent {
+class FlutterDemo extends StatefulComponent {
+  @override
```

Comment:
Aside: curious what we want the tools to do.

If I use the annotation, I get a warning if my superclass doesn't contain the method

![screen shot 2016-01-12 at 10 55 40 am](https://cloud.githubusercontent.com/assets/5479/12273524/184d7c80-b91b-11e5-9847-46fe61d7a4be.png)

Which can help when code is refactored.